tests/CacheCompat/CommonCache.Test.Unit/CommonCache.Test.Unit.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
```

Comment:
I didnt see anything in the component governance errors related to this? Was this a problem too?

Reply:
![image](https://user-images.githubusercontent.com/12273384/176397894-959f2f8a-5918-4c4c-95e3-1f0b2d858224.png)